sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/ShareTokenIntent.java

```diff
@@ -8,7 +8,11 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import java.util.Collection;
 
-/** The request intent for whether the file should be backed up. */
+/**
+ * The request intent for whether the file should be backed up. Possible values are: {@link ShareTokenIntent#BACKUP} -
```

Comment:
Same. Really not sure why the actual string enum class that has all the values publicly declared needs them to also be declared in a docstring that will likely go out of date.